data/systemd/snap-repair.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Timer to automatically fetch and run repair assertions
```

Comment:
On classic it will just exit(0). The reason it is there at all on classic is mostly because we use the exactly same packaging on classic and core. We could change that of course if its something we feel strongly about (it will be a bit of work).